docs/reference/stream/zchannel/index.md

```diff
@@ -44,9 +44,9 @@ case class ZPipeline[-R, +E, -In, +Out] (
 
 So we can say that:
 
-- `ZStream[R, E, A]` is a channel that uses `R` as its environment, produce `Chunk[A]` to its output port, can terminate with `Any` success value or can terminate with a failure of type `E`.
+- `ZStream[R, E, A]` is a channel that uses `R` as its environment, produce `Chunk[A]` to its output port, can terminate with a success value of type `A`  or can terminate with a failure of type `E`.
```

Comment:
This is not correct.

Reply:
could you give some adivce on this, how about `can produce elements of type A or can terminate iwth a failure of type E` ?

Reply:
`ZStream[R, E, A] is a channel that requires an environment `R`, emits elements of type `Chunk[A]`, and terminates, if at all, with either an error of type `E` or a done value of type `Any`.

Reply:
updated.